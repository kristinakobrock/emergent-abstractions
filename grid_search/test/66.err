  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 27.27it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 23.59it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 25.83it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 25.58it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 25.36it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 24.65it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 24.90it/s] 48%|############################################1                                               | 24/50 [00:00<00:00, 26.17it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 26.00it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 28.48it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 26.42it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 25.11it/s] 82%|###########################################################################4                | 41/50 [00:01<00:00, 27.42it/s] 88%|################################################################################9           | 44/50 [00:01<00:00, 25.94it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 25.98it/s]100%|############################################################################################| 50/50 [00:01<00:00, 26.35it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 29.58it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 24.89it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 23.51it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 23.05it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.37it/s]
