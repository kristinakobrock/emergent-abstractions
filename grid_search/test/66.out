# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7069931030273438, "acc": 0.5910044312477112, "length": 3.500953197479248, "mode": "train", "epoch": 1}
{"loss": 0.6758928298950195, "acc": 0.6031250357627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6765713095664978, "acc": 0.5896593928337097, "length": 3.27583646774292, "mode": "train", "epoch": 2}
{"loss": 0.6590408086776733, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6676987409591675, "acc": 0.6282609701156616, "length": 2.6656408309936523, "mode": "train", "epoch": 3}
{"loss": 0.6709777116775513, "acc": 0.6173827648162842, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6600945591926575, "acc": 0.6304458379745483, "length": 1.9764783382415771, "mode": "train", "epoch": 4}
{"loss": 0.6581571698188782, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6580567359924316, "acc": 0.631758451461792, "length": 1.5240733623504639, "mode": "train", "epoch": 5}
{"loss": 0.6591648459434509, "acc": 0.6298828125, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6565046906471252, "acc": 0.6319721937179565, "length": 1.2207216024398804, "mode": "train", "epoch": 6}
{"loss": 0.6583136320114136, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6561511158943176, "acc": 0.6320273280143738, "length": 1.1400705575942993, "mode": "train", "epoch": 7}
{"loss": 0.6577775478363037, "acc": 0.6292969584465027, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6547776460647583, "acc": 0.631942868232727, "length": 1.1093871593475342, "mode": "train", "epoch": 8}
{"loss": 0.6580851078033447, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6558443903923035, "acc": 0.6320708990097046, "length": 1.0988599061965942, "mode": "train", "epoch": 9}
{"loss": 0.6577991247177124, "acc": 0.6292968392372131, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6555672883987427, "acc": 0.6318846940994263, "length": 1.0942147970199585, "mode": "train", "epoch": 10}
{"loss": 0.6584113836288452, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6552003026008606, "acc": 0.63197922706604, "length": 1.100569486618042, "mode": "train", "epoch": 11}
{"loss": 0.6579035520553589, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6555407643318176, "acc": 0.6317757368087769, "length": 1.1092450618743896, "mode": "train", "epoch": 12}
{"loss": 0.6578507423400879, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6533023118972778, "acc": 0.6321266889572144, "length": 1.137961983680725, "mode": "train", "epoch": 13}
{"loss": 0.6579908132553101, "acc": 0.6292969584465027, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6545514464378357, "acc": 0.6319830417633057, "length": 1.2615915536880493, "mode": "train", "epoch": 14}
{"loss": 0.6581674814224243, "acc": 0.6294922232627869, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6515738368034363, "acc": 0.632573127746582, "length": 1.536231279373169, "mode": "train", "epoch": 15}
{"loss": 0.6600327491760254, "acc": 0.6292969584465027, "length": 1.0, "mode": "test", "epoch": 15}
{"loss": 0.643424928188324, "acc": 0.6346541047096252, "length": 1.6812695264816284, "mode": "train", "epoch": 16}
{"loss": 0.6431260108947754, "acc": 0.6385239362716675, "length": 1.56640625, "mode": "test", "epoch": 16}
{"loss": 0.6320586204528809, "acc": 0.6444411873817444, "length": 2.064148187637329, "mode": "train", "epoch": 17}
{"loss": 0.6258053183555603, "acc": 0.6517730355262756, "length": 2.16015625, "mode": "test", "epoch": 17}
{"loss": 0.6195897459983826, "acc": 0.6603831052780151, "length": 2.329887628555298, "mode": "train", "epoch": 18}
{"loss": 0.6199783086776733, "acc": 0.658416748046875, "length": 2.11328125, "mode": "test", "epoch": 18}
{"loss": 0.6025349497795105, "acc": 0.676148533821106, "length": 2.241295099258423, "mode": "train", "epoch": 19}
{"loss": 0.6092653274536133, "acc": 0.6737762689590454, "length": 2.359375, "mode": "test", "epoch": 19}
{"loss": 0.5934813618659973, "acc": 0.6853592395782471, "length": 2.306121349334717, "mode": "train", "epoch": 20}
{"loss": 0.6095796823501587, "acc": 0.6686416864395142, "length": 2.453125, "mode": "test", "epoch": 20}
{"loss": 0.5788092017173767, "acc": 0.6981393098831177, "length": 2.59521222114563, "mode": "train", "epoch": 21}
{"loss": 0.6104646325111389, "acc": 0.6711686253547668, "length": 2.34765625, "mode": "test", "epoch": 21}
{"loss": 0.569736897945404, "acc": 0.7053818106651306, "length": 2.478482246398926, "mode": "train", "epoch": 22}
{"loss": 0.6020231246948242, "acc": 0.6808533072471619, "length": 2.59375, "mode": "test", "epoch": 22}
{"loss": 0.5648149251937866, "acc": 0.7100419998168945, "length": 2.5312845706939697, "mode": "train", "epoch": 23}
{"loss": 0.6033061742782593, "acc": 0.6811431646347046, "length": 2.44140625, "mode": "test", "epoch": 23}
{"loss": 0.5633443593978882, "acc": 0.7107527256011963, "length": 2.519402503967285, "mode": "train", "epoch": 24}
{"loss": 0.6093812584877014, "acc": 0.6849472522735596, "length": 2.41796875, "mode": "test", "epoch": 24}
{"loss": 0.5584830641746521, "acc": 0.7139569520950317, "length": 2.47698974609375, "mode": "train", "epoch": 25}
{"loss": 0.6000308394432068, "acc": 0.6809844970703125, "length": 2.7109375, "mode": "test", "epoch": 25}
{"loss": 0.5504435300827026, "acc": 0.7189689874649048, "length": 2.5725972652435303, "mode": "train", "epoch": 26}
{"loss": 0.5908330082893372, "acc": 0.6877380013465881, "length": 2.546875, "mode": "test", "epoch": 26}
{"loss": 0.5450522303581238, "acc": 0.7193174958229065, "length": 2.5546069145202637, "mode": "train", "epoch": 27}
{"loss": 0.5872661471366882, "acc": 0.6858062744140625, "length": 2.83984375, "mode": "test", "epoch": 27}
{"loss": 0.5431946516036987, "acc": 0.7239761352539062, "length": 2.6526684761047363, "mode": "train", "epoch": 28}
{"loss": 0.5778927803039551, "acc": 0.6975952386856079, "length": 2.73828125, "mode": "test", "epoch": 28}
{"loss": 0.5366286039352417, "acc": 0.7271077632904053, "length": 2.747309684753418, "mode": "train", "epoch": 29}
{"loss": 0.5732771754264832, "acc": 0.7030715942382812, "length": 2.95703125, "mode": "test", "epoch": 29}
{"loss": 0.532380998134613, "acc": 0.7332795262336731, "length": 2.8168718814849854, "mode": "train", "epoch": 30}
{"loss": 0.5645996332168579, "acc": 0.7123535871505737, "length": 2.9140625, "mode": "test", "epoch": 30}
{"loss": 0.521220862865448, "acc": 0.7402894496917725, "length": 2.8761041164398193, "mode": "train", "epoch": 31}
{"loss": 0.559981107711792, "acc": 0.7219955325126648, "length": 3.14453125, "mode": "test", "epoch": 31}
{"loss": 0.5059460997581482, "acc": 0.7546390891075134, "length": 3.0625975131988525, "mode": "train", "epoch": 32}
{"loss": 0.5547268390655518, "acc": 0.7287291884422302, "length": 3.09765625, "mode": "test", "epoch": 32}
{"loss": 0.4944155514240265, "acc": 0.7664394974708557, "length": 3.0935022830963135, "mode": "train", "epoch": 33}
{"loss": 0.5583046674728394, "acc": 0.7358382940292358, "length": 3.23046875, "mode": "test", "epoch": 33}
{"loss": 0.4869496524333954, "acc": 0.7695894241333008, "length": 3.113776206970215, "mode": "train", "epoch": 34}
{"loss": 0.5422071218490601, "acc": 0.7402756214141846, "length": 3.28515625, "mode": "test", "epoch": 34}
{"loss": 0.4846966564655304, "acc": 0.7695515751838684, "length": 3.232632637023926, "mode": "train", "epoch": 35}
{"loss": 0.5355358719825745, "acc": 0.7485671639442444, "length": 3.21875, "mode": "test", "epoch": 35}
{"loss": 0.4748242497444153, "acc": 0.7764879465103149, "length": 3.1433942317962646, "mode": "train", "epoch": 36}
{"loss": 0.5195573568344116, "acc": 0.7462509870529175, "length": 3.17578125, "mode": "test", "epoch": 36}
{"loss": 0.4706231653690338, "acc": 0.7781976461410522, "length": 3.1428003311157227, "mode": "train", "epoch": 37}
{"loss": 0.5214210748672485, "acc": 0.7475464344024658, "length": 3.17578125, "mode": "test", "epoch": 37}
{"loss": 0.46485772728919983, "acc": 0.7787598967552185, "length": 3.176063060760498, "mode": "train", "epoch": 38}
{"loss": 0.5129928588867188, "acc": 0.7525177597999573, "length": 3.06640625, "mode": "test", "epoch": 38}
{"loss": 0.462758332490921, "acc": 0.7814615964889526, "length": 3.0937280654907227, "mode": "train", "epoch": 39}
{"loss": 0.5106462240219116, "acc": 0.7526611685752869, "length": 3.16015625, "mode": "test", "epoch": 39}
{"loss": 0.45854321122169495, "acc": 0.7830237150192261, "length": 3.1557583808898926, "mode": "train", "epoch": 40}
{"loss": 0.5083397030830383, "acc": 0.7541717886924744, "length": 3.05859375, "mode": "test", "epoch": 40}
{"loss": 0.45522889494895935, "acc": 0.7852586507797241, "length": 3.165177345275879, "mode": "train", "epoch": 41}
{"loss": 0.5135139226913452, "acc": 0.7530166506767273, "length": 2.99609375, "mode": "test", "epoch": 41}
{"loss": 0.45243754982948303, "acc": 0.7869291305541992, "length": 3.0945756435394287, "mode": "train", "epoch": 42}
{"loss": 0.5030582547187805, "acc": 0.756309449672699, "length": 3.265625, "mode": "test", "epoch": 42}
{"loss": 0.45152536034584045, "acc": 0.7856109142303467, "length": 3.1855738162994385, "mode": "train", "epoch": 43}
{"loss": 0.500166654586792, "acc": 0.760589599609375, "length": 3.05859375, "mode": "test", "epoch": 43}
{"loss": 0.4473116397857666, "acc": 0.7872393131256104, "length": 3.136934757232666, "mode": "train", "epoch": 44}
{"loss": 0.5010851621627808, "acc": 0.7577926516532898, "length": 3.09375, "mode": "test", "epoch": 44}
{"loss": 0.44896793365478516, "acc": 0.7872896790504456, "length": 3.1598944664001465, "mode": "train", "epoch": 45}
{"loss": 0.49441251158714294, "acc": 0.764291524887085, "length": 3.15625, "mode": "test", "epoch": 45}
{"loss": 0.44610118865966797, "acc": 0.7855555415153503, "length": 3.177290678024292, "mode": "train", "epoch": 46}
{"loss": 0.5016804337501526, "acc": 0.7577987313270569, "length": 3.08984375, "mode": "test", "epoch": 46}
{"loss": 0.444094181060791, "acc": 0.7866420149803162, "length": 3.1669018268585205, "mode": "train", "epoch": 47}
{"loss": 0.4999822974205017, "acc": 0.7598403692245483, "length": 3.1484375, "mode": "test", "epoch": 47}
{"loss": 0.4431218206882477, "acc": 0.7865419983863831, "length": 3.1977319717407227, "mode": "train", "epoch": 48}
{"loss": 0.48843902349472046, "acc": 0.7648804187774658, "length": 3.15625, "mode": "test", "epoch": 48}
{"loss": 0.44546565413475037, "acc": 0.7884352207183838, "length": 3.1812641620635986, "mode": "train", "epoch": 49}
{"loss": 0.49500569701194763, "acc": 0.7629410028457642, "length": 3.15234375, "mode": "test", "epoch": 49}
{"loss": 0.43983978033065796, "acc": 0.7898973226547241, "length": 3.2136454582214355, "mode": "train", "epoch": 50}
{"loss": 0.4879601001739502, "acc": 0.7649490237236023, "length": 3.1640625, "mode": "test", "epoch": 50}
{"loss": 0.44097980856895447, "acc": 0.7907148599624634, "length": 3.155440092086792, "mode": "train", "epoch": 51}
{"loss": 0.4951750338077545, "acc": 0.7632735967636108, "length": 3.21875, "mode": "test", "epoch": 51}
{"loss": 0.4377274513244629, "acc": 0.792089581489563, "length": 3.2055091857910156, "mode": "train", "epoch": 52}
{"loss": 0.4858947992324829, "acc": 0.7675140500068665, "length": 3.16796875, "mode": "test", "epoch": 52}
{"loss": 0.43515655398368835, "acc": 0.7933698892593384, "length": 3.185896158218384, "mode": "train", "epoch": 53}
{"loss": 0.4855101704597473, "acc": 0.7667160630226135, "length": 3.19140625, "mode": "test", "epoch": 53}
{"loss": 0.4322347044944763, "acc": 0.7930695414543152, "length": 3.2197012901306152, "mode": "train", "epoch": 54}
{"loss": 0.49060314893722534, "acc": 0.7680526971817017, "length": 3.1640625, "mode": "test", "epoch": 54}
{"loss": 0.4301779568195343, "acc": 0.7969853281974792, "length": 3.162170648574829, "mode": "train", "epoch": 55}
{"loss": 0.4926786422729492, "acc": 0.7646331787109375, "length": 3.234375, "mode": "test", "epoch": 55}
{"loss": 0.4276508688926697, "acc": 0.7978405356407166, "length": 3.223548412322998, "mode": "train", "epoch": 56}
{"loss": 0.4969736337661743, "acc": 0.7646346688270569, "length": 3.1328125, "mode": "test", "epoch": 56}
{"loss": 0.4265998303890228, "acc": 0.7982323169708252, "length": 3.170100688934326, "mode": "train", "epoch": 57}
{"loss": 0.4882648289203644, "acc": 0.7663513422012329, "length": 3.2734375, "mode": "test", "epoch": 57}
{"loss": 0.4280978739261627, "acc": 0.7984190583229065, "length": 3.2174086570739746, "mode": "train", "epoch": 58}
{"loss": 0.5003082752227783, "acc": 0.7648223638534546, "length": 3.1875, "mode": "test", "epoch": 58}
{"loss": 0.430374413728714, "acc": 0.797670304775238, "length": 3.218312978744507, "mode": "train", "epoch": 59}
{"loss": 0.4815182685852051, "acc": 0.7702530026435852, "length": 3.21484375, "mode": "test", "epoch": 59}
{"loss": 0.42384031414985657, "acc": 0.7984288930892944, "length": 3.2216169834136963, "mode": "train", "epoch": 60}
{"loss": 0.4873225688934326, "acc": 0.7660506367683411, "length": 3.203125, "mode": "test", "epoch": 60}
test accuracy: 0.6821188926696777
